docs/docs/workflows/triggers.md

```diff
@@ -0,0 +1,125 @@
+    <img style={{padding: '10px'}} className="screenshot-full" src="/img/workflows/triggers/env.png" alt="Triggers" />
+  </div>
+
+- Find the API endpoint URL in the **Endpoint** field. You can use this URL to send a POST request to trigger the workflow. You can also click on the **Copy** button to copy the URL to the clipboard. You can either select `Copy URL` or `Copy as cURL` from the dropdown menu. The `Copy as cURL` option copies the URL as a cURL command which will include details such as the `API token` and `Environment`. An example of the Endpoint URL is as follows:
+
+  ```
+  http://{TOOLJET_HOST}/api/v2/webhooks/workflows/:id/trigger
+  ```
+
+  <div style={{textAlign: 'center'}}>
+    <img style={{padding: '10px'}} className="screenshot-full" src="/img/workflows/triggers/copy.png" alt="Triggers" />
+  </div>
+
+- The API token is used to authenticate the request. You can find the API token in the **API Token** field. You can also click on the **Copy** button to copy the API token to the clipboard.
+
+  <div style={{textAlign: 'center'}}>
+    <img style={{padding: '10px'}} className="screenshot-full" src="/img/workflows/triggers/token.png" alt="Triggers" />
+  </div>
+
+- Parameters can be passed to the workflow using the **Parameters** field. The parameter `key` and their `type` can be specified in the **Parameters** field. For example, if you want to pass the `name` and `age` parameters to the workflow using the webhook triggers, you can set the **Parameters** field as follows:
+
+  ```json
+  "name": "string",
+  "age": "number"
+  ```
+
+  <div style={{textAlign: 'center'}}>
+    <img style={{padding: '10px'}} className="screenshot-full" src="/img/workflows/triggers/params.png" alt="Triggers" />
+  </div>
+
+- The **Test JSON parameters** field can be used to test the webhook trigger. You can enter the parameter values in the **Test JSON parameters** field and click on the **Run** button to test the webhook trigger. The workflow will be executed with the parameter values specified in the **Test JSON parameters** field.
+
+  ```json
+  {
+    "name": "John Doe",
+    "age": 30
+  }
+  ```
+
+  <div style={{textAlign: 'center'}}>
+    <img style={{padding: '10px'}} className="screenshot-full" src="/img/workflows/triggers/test.png" alt="Triggers" />
+  </div>
+
+### Restrictions on usage of webhook triggers
+
+There are certain restrictions on the usage of webhook triggers that are configurable, both at the instance level and at the workspace level depending on the license. The restrictions are as follows:
+
+- Number of executions per month
+- Number of executions per day
+- Number of parallel executions
+- Execution time per workflow
+
+For limiting parallel executions, the following environment variables can be used:
+
+| Environment variable | Value | Description |
+| -------------------- | ----- | ----------- |
+| WORKFLOW_WEBHOOK_THROTTLE_TTL | 60000 | Time in milliseconds for which the throttling will be applied |
```

Comment:
This env var has been updated
WEBHOOK_THROTTLE_TTL - Time in milliseconds for the webhook requests to live
WEBHOOK_THROTTLE_LIMIT - Maximum number of requests within the TTL that will be throttled